crypto/zkp/primitives/representationproofs/representation.go

```diff
@@ -0,0 +1,165 @@
+/*
```

Comment:
I just realised that `crypto/zkp/primitives/representationproofs/representation.go` and `crypto/representationproofs/representation.go` are exactly the same (except the first one has a license header); Is this is a mistake? If yes, please use just one and remove the other; as for the comments: please combine my review comments for both files (they might be a little scattered) and apply them to this one file.